cmd/bisync/operations.go

```diff
@@ -224,7 +224,7 @@ func (b *bisyncRun) runLocked(octx context.Context) (err error) {
 		return err
 	}
 
-	fs.Infof(nil, "Synching Path1 %s with Path2 %s", quotePath(path1), quotePath(path2))
+	fs.Logf(nil, "Synching Path1 %s with Path2 %s", quotePath(path1), quotePath(path2))
```

Comment:
Note that this change would affect all runs, not just dry-runs. I don't necessarily mind it (I usually use `-v` anyway), but I wonder if some users would find it unnecessary?

Reply:
I'll change this back so as to not make out of scope changes.